spec/helpers/application_helper_spec.rb

```diff
@@ -41,4 +41,78 @@
       expect(helper.library_name('NOSUCHLIBRARY')).to eq 'NOSUCHLIBRARY'
     end
   end
+
+  describe '#render_checkout_status' do
+    let(:checkout) do
+      instance_double(
+        Checkout,
+        recalled?: false,
+        overdue?: false,
+        lost?: false,
+        claimed_returned?: false,
+        accrued: 0
+      )
+    end
+    let(:content) { Capybara.string(helper.render_checkout_status(checkout)) }
+
+    context 'when a recalled item has accrued fines' do
+      before do
+        allow(checkout).to receive_messages(recalled?: true, accrued: 15)
+      end
+
+      it 'renders the right html' do
+        expect(content).to have_css('.text-recalled', text: 'Recalled $15').and(have_css('.sul-icons'))
+      end
+    end
+
+    context 'when a recalled item has no accrued fines' do
+      before do
+        allow(checkout).to receive_messages(recalled?: true)
+      end
+
+      it 'renders the right html' do
+        expect(content).to have_css('.text-recalled', text: 'Recalled').and(have_css('.sul-icons'))
+      end
+    end
+
+    context 'when an item is claimed returned' do
+      before do
+        allow(checkout).to receive_messages(overdue?: true, lost?: true, claimed_returned?: true, accrued: 666)
```

Comment:
@tallenaz and @cbeer, just want to confirm my understanding re: `Processing claim` state. 
- I confirmed in #131 and our labels and content doc that it is not supposed to display an icon.
- The item has an orange color because the item is technically still overdue, but no icon is displayed because there's no further action a patron can / should take.

<img width="1139" alt="processing claim record with orange background color" src="https://user-images.githubusercontent.com/5402927/61496914-9d2f4f00-a972-11e9-9fd6-730990ea3916.png">

Is that right?

Reply:
That's my understanding; in any case, if it's not supposed to be orange, that'll be separate from the status value.

Reply:
It is correct as shown.

Reply:
Thanks for the clarification!